probedata.go

```diff
@@ -102,6 +103,35 @@ type StreamDisposition struct {
 	AttachedPic     int `json:"attached_pic"`
 }
 
+// Chapters is a json data structure to represent chapters.
+type Chapter struct {
+	ID               int         `json:"id"`
+	TimeBase         string      `json:"time_base"`
+	StartTimeSeconds float64     `json:"start_time,string"`
+	EndTimeSeconds   float64     `json:"end_time,string"`
+	Tags             *ChapterTags `json:"tags"`
+}
+
+// ChapterTags is a json data structure to represent metadata of a chapter.
+type ChapterTags struct {
+	Title string `json:"title"`
+}
+
+// StartTime returns the start time of the chapter as a time.Duration
+func (c *Chapter) StartTime() (duration time.Duration) {
```

Comment:
I dont think there is a need to declare the return var, so:
```suggestion
func (c *Chapter) StartTime() time.Duration {
```